.github/workflows/github-runner-image-push.yml

```diff
@@ -0,0 +1,105 @@
+# This workflow will trigger when it detects that a new github runner image was built in the
+# github-runner-image-build.yml workflow. It will download the artifacts built from that workflow
+# and push it to an ECR in each region
+name: Push Github Runner Image
+
+on:
+  workflow_run:
+    workflows:
+      - "Build Github Runner Image"
```

Comment:
So this runs after the other workflow or does it trigger when the other triggers? ie. are they parallelized or serialized?

Reply:
They are serialized. It get triggered when the "Build Github Runner Image" workflow run is completed on the main branch